hotel-reservation-system/src/components/shared/BaseCard.js

```diff
@@ -34,23 +33,35 @@ export default function BaseCard({
 }) {
   const classes = useStyles();
   const [reservationDialogOpen, setReservationDialogOpen] = useState(false);
-  const history = useHistory();
+  let islogged = useSelector((state) => state.isLogged);
```

Comment:
Please try to use 'const' instead of 'let' if you are not going to mutate the object

Reply:
replaced with const